### PR TITLE
thesis.md: Fix typo in file

### LINKED ISSUE #411: Fix typo in thesis.md
There is typo in file `_faq/thesis.md`, `recommend` is misspelled as `recoment`
label: difficulty/newcomer
Opened by @Monal5031 at [Gitter](https://gitter.im/coala/coala-bears?at=5a01f04ee44c43700acb6e9d)

### PR DESCRIPTION
**Pull request by Lucifer24**
Fixed typo in thesis.md file,
`recommend` was misspelled as
`recoment`

Closes #411

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: _faq/thesis.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ question: "I want to write a thesis with coala!"
 ---
 Great! This is the right place. Check out the other questions and most
 importantly the projects page.
-If you have own ideas we recoment you submit them to
-[our projects list](https://github.com/coala/projects/blob/master/data/projects.js)
```
Comment (Monal5031) [inline _faq/thesis.md]: Please revert this change, it is not a part of this issue.

**Comment by Monal5031** [comment]
@Lucifer24 Please read the newcomers guide first before starting to contribute, it is mentioned there that how do we write a commit message.
A good commit message is something like:
```
filename: Head-log

Short description of changes

Closes #{issue-numer}
```
Please amend your commit message in this way.
If you need any help please ping on our gitter channel, someone will surely help 😄

**Comment by Monal5031** [comment]
ack dcc7938

**Comment by jayvdb** [comment]
@rultor merge
